README.md

```diff
@@ -2,62 +2,10 @@
 
 [![Build Status](https://travis-ci.org/tidepool-org/chrome-uploader.png)](https://travis-ci.org/tidepool-org/chrome-uploader)
 
-These notes were updated on Tuesday, December 30, 2014.
```

Comment:
I don't care too much, but the reason I put such things in is that people can easily know how current the notes are.